### DISCUSSION TITLE
Bulk import power feed cables

### DISCUSSION THREAD

**Discussion by lucix-mich** [netbox-community/netbox#6853] [Q&A]
Hello all,

How would one go about bulk importing power feed cables? While initially looking at this, I couldn't figure out what to specifiy for the required fields on the power feed side of the cable. Required are: side_b_device,side_b_type,side_b_name.

I succeeded with side_b_type (dcim.powerfeed), but not side_b_device. What values does the device take if it represents a power feed? Error I get is:

`Row 1 side_b_device: Object not found.`

I used the name of a power panel, since power feeds are children of power panels, and the device_b_name would represent the name of the feed.

I also tried using the name of the power feed itself, but that doesn't work neither, and make sense since the name of a power feed in itself is not unique.

Thanks and cheers

Michel

**Comment by candlerb** [comment]
Normally I run power cables between powerport and poweroutlet.

I haven't tried with powerfeed, and to me this sounds similar to the problem reported in #6840.  Powerfeeds are to power as circuitterminations are to interface - a sort of "virtual endpoint" that you can connect a cable to.

I don't see an obvious solution, since as you've observed, a powerfeed is not presented on a "device" (nor is a circuit termination).  It would require raising as a feature request.

Alternatively, you could write a custom script which accepts a CSV of power panels and powerports, and creates cables between them.

**Comment by lucix-mich** [comment]
Thanks @candlerb .

But then this seems like a bug or at least missing feature, because power cables connecting a "pdu" device to a powerfeed are listed among all other cables, so this suggest they can be bulk imported as any other cable.

And the same would probably be the case for circuits (haven't tried this myself).

Do you have a hint at where the import code is located? I might take a look.

Thanks and cheers
